test/blocks/nav-marquee/mocks/body.html

```diff
@@ -0,0 +1,108 @@
+<div class="nav-marquee light medium">
+  <div class="marquee light">
+    <div class="foreground">
+      <div class="text">
+        <div>
+          <h2 id="heading-xl-bold-3645-lorem-ipsum" class="heading-xl">Heading XL Bold (36/45) Lorem ipsum</h2>
+          <p class="body-m">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+            labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation.</p>
+          <p class="action-area"><a href="#undefined" class="con-button outline button-l">Lorem ipsum</a> <a
+              href="https://www.adobe.com/" class="con-button blue button-l">Call to action</a></p>
+          <p class="supplemental-text body-xl">
+            <picture>
+              <source type="image/webp"
+                srcset="./media_1170eda005e78c9b73449a7c09ecb2a49290775ef.png?width=2000&amp;format=webply&amp;optimize=medium"
+                media="(min-width: 600px)">
+              <source type="image/webp"
+                srcset="./media_1170eda005e78c9b73449a7c09ecb2a49290775ef.png?width=750&amp;format=webply&amp;optimize=medium">
+              <source type="image/png"
+                srcset="./media_1170eda005e78c9b73449a7c09ecb2a49290775ef.png?width=2000&amp;format=png&amp;optimize=medium"
+                media="(min-width: 600px)">
+              <img loading="eager" alt="" type="image/png"
+                src="./media_1170eda005e78c9b73449a7c09ecb2a49290775ef.png?width=750&amp;format=png&amp;optimize=medium"
+                width="750" height="424">
+            </picture>
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="content-table">
+    <div class="toc-container">
+      <p class="toc-title">This is the title of the section</p>
+      <nav aria-label="Table of contents">
+        <ul class="toc-list">
+          <li class="toc-item">
```

Comment:
I am not able to TAB through the toc items, this breaks keyboard navigation / accessibility patterns.

Reply:
Added tabindex